features/steps/formatting_steps.rb

```diff
@@ -251,7 +251,7 @@
 end
 
 Then(/^I should see a red error message$/) do
-  run_output.should include(red("⌦  " + SAMPLE_PODS_ERROR.gsub('error: ', '')))
+  run_output.should include(red("❌  " + SAMPLE_PODS_ERROR.gsub('error: ', '')))
```

Comment:
Or this one that Xcode is using: 
<img width="30" alt="screen shot 2015-10-05 at 3 38 10 pm" src="https://cloud.githubusercontent.com/assets/627285/10295691/1f8d61be-6b77-11e5-9b9e-a0e6b76a0cd2.png">